components/automate-ui/src/app/pages/api-token/details/api-token-details.component.ts

```diff
@@ -1,14 +1,14 @@
 import { Component, OnInit, OnDestroy } from '@angular/core';
 import { FormBuilder, FormGroup, Validators, FormControl } from '@angular/forms';
 import { Store, select } from '@ngrx/store';
-import { isEmpty, identity, xor } from 'lodash/fp';
 import { combineLatest, Subject } from 'rxjs';
 import { filter, map, pluck, takeUntil } from 'rxjs/operators';
+import { isEmpty, identity, xor } from 'lodash/fp';
```

Comment:
do we have a rule of thumb for when to use lodash vs. rxjs operators? they seem to have some overlap (at least name wise)

Reply:
You are quite right; they definitely have name overlap, which is likely by design because they operate on different domains but perform similar functions. That is, lodash/fp operators work on collections of objects whereas rxjs operators work on streams of Observables.